doc/en/developer/source/quickstart/intellij.rst

```diff
@@ -72,4 +72,11 @@ Access GeoServer front page
 ---------------------------
 
 * After a few seconds, GeoServer should be accessible at: `<http://localhost:8080/geoserver>`_
-* The default ``admin`` password is ``geoserver``.
\ No newline at end of file
+* The default ``admin`` password is ``geoserver``.
+
+Run GeoServer from Intellij on Windows
+---------------------------
+
+#. Add bash to your Windows environment path and restart Intellij.  
+#. Rebuild the security-tests project in the security module.  Right click on the security-tests project and click Rebuild.  Otherwise,  there will be symbol errors such as "cannot find symbol AbstractUserGroupServiceTest".
+#. Using wcs1_1 as the working directory, run a ``mvn clean install``.  Some of the generated code from this project does not get include in the build.  This causes errors like "cannot find symbol class ASTAxisId"
```

Comment:
This is due to some code being generated from a javacc spec. Not windows specific, would happen on any platform if the maven build is not run before.  I think it can be done from IntelliJ too, right click on the project, "maven", "generate sources and update folders". I don't know why it does not get done automatically, or if there is a way to have that happen, in a normal maven build it's just part of the build sequence.